mock/src/main/java/com/tngtech/keycloakmock/impl/helper/BasicTokenHelper.java

```diff
@@ -0,0 +1,29 @@
+package com.tngtech.keycloakmock.impl.helper;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.Nonnull;
+
+/** Helper to handle Keycloak base64 basic token. */
+public class BasicTokenHelper {
+
+  @Nonnull
+  public Map<String, String> parseToken(@Nonnull String token) {
```

Comment:
Please return an actual business object (and throw some Exception if the token is not valid).